packages/vizbuilder/src/helpers/chartconfig.js

```diff
@@ -0,0 +1,334 @@
+      ...topojsonConfig,
+      ...flags.chartConfig
+    };
+
+    if (isActive) {
+      config.colorScaleConfig = {
+        scale: "jenks"
+      };
+    }
+
+    return config;
+  },
+  histogram(commonConfig, query, flags) {
+    const config = this.barchart(commonConfig, query, flags);
+
+    return {
+      ...config,
+      // title: `Histogram of ${measureName} by ${drilldownName}`,
+      groupPadding: 0,
+      ...flags.chartConfig
+    };
+  },
+  lineplot(commonConfig, query, flags) {
+    const {timeDrilldown, drilldown, measure, moe} = query;
+
+    const drilldownName = timeDrilldown.name;
+    const measureName = measure.name;
+
+    const config = {
+      ...commonConfig,
+      title: `Line plot of ${measureName} by ${drilldownName}`,
+      discrete: "x",
+      groupBy: drilldown.hierarchy.levels[1].name,
+      x: drilldownName,
+      xConfig: {title: drilldownName},
+      y: measureName,
+      yConfig: {title: measureName},
+      ...flags.chartConfig
+    };
+
+    if (moe) {
+      const moeName = moe.name;
+
+      config.confidence = [
+        d => d[measureName] - d[moeName],
+        d => d[measureName] + d[moeName]
+      ];
+    }
+
+    return config;
+  },
+  pie(commonConfig, query, flags) {
+    return this.donut(commonConfig, query, flags);
+  },
+  stacked(commonConfig, query, flags) {
+    const config = this.lineplot(commonConfig, query, flags);
+
+    const drilldownName = query.drilldown.name;
+    const measureName = query.measure.name;
+
+    return {
+      ...config,
+      title: `Stacked area chart of ${measureName} by ${drilldownName}`,
+      ...flags.chartConfig
+    };
+  },
+  treemap(commonConfig, query, flags) {
+    const {drilldown, measure} = query;
+
+    const drilldownName = drilldown.name;
+    const measureName = measure.name;
+
+    const levels = drilldown.hierarchy.levels;
+    const ddIndex = levels.indexOf(drilldown);
+
+    const config = {
+      ...commonConfig,
+      title: `Treemap of ${measureName} by ${drilldownName}`,
+      groupBy: levels.slice(1, ddIndex + 1).map(lvl => lvl.name),
+      ...flags.chartConfig
+    };
+
+    return config;
+  }
+};
+
+/**
+ * Generates an array with valid config objects, depending on the type of data
+ * retrieved and the current user defined parameters, to use in d3plus charts.
+ * @param {CreateChartConfigParams} param0 The object containing the parameters
+ * @returns {CreateChartConfigResult[]}
+ */
+export default function createChartConfig({
+  activeType,
+  members,
+  query,
+  topojson,
+  userConfig,
+  visualizations,
+  year
+}) {
+  const availableKeys = new Set(Object.keys(members));
+  const availableCharts = new Set(
+    activeType ? [activeType] : visualizations
+  );
+
+  const drilldownName = query.drilldown.name;
+  const measureName = query.measure.name;
+  const getMeasureName = d => d[measureName];
+
+  const commonConfig = {
+    height: activeType ? 500 : 400,
+    legend: false,
+
```

Comment:
When viewing tooltips, I'm seeing the drilldown type and the value (ie. "State California"). It should just be the value. Maybe the `commonConfig` should have `label: dimensionName`?

Reply:
applied in f909c70bed94a0d7bd8c57d085cc500e6c140875